templates/custom/flake.nix

```diff
@@ -21,8 +21,6 @@
         formatter = pkgs.nixpkgs-fmt;
 
         devShells.default = pkgs.mkShell {
-          inputsFrom = [ inputs'.holonix.devShells ];
```

Comment:
This was buggy, should have been 

```diff
- inputsFrom = [ inputs'.holonix.devShells ];
+ inputsFrom = [ inputs'.holonix.devShells.default ];
```

The idea was to keep this Holonix flexible, so I prefer removing the `inputsFrom` so that people can choose what to include in their dev shell

Reply:
I see, will remove this from the scaffolded nix flake, had actually modified it to the snippet below.

So ultimately we continue to have the holochain nix packages explicitly defined except for `rust` from what I understood from this PR and from the conversation in https://github.com/holochain/scaffolding/issues/365

```nix
inputsFrom = [ inputs'.holonix.devShells.default ];
```